Automattic/wp-calypso
# Design Picker: Fix the style of premium badge is broken

#### Proposed Changes

* The issue is introduced from https://github.com/Automattic/wp-calypso/pull/67539 as it imports the `choose-a-domain` component into the stepper framework. That component contains the `premium-badge` component and it defines `div.premium-badge` classname to override the styles
* The changes is to wrap the `div.premium-badge` classname so that it won't override the any component with `premium-badge` classname

#### Testing Instructions



* Go to `/setup?siteSlug=<your_site>`
* Always click "Continue" until we land on the Design Picker
* Verify whether the style of the premium badge is fixed on the Design Picker step

#### Pre-merge Checklist





Related to p1668430568531859-slack-C048CUFRGFQ
